src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/CallbackRespHelper.java

```diff
@@ -13,6 +13,7 @@
 
 import java.util.List;
 
+@SuppressWarnings({"PMD.ClassWithOnlyPrivateConstructorsShouldBeFinal"})
```

Comment:
You can just write final keyword in class rather than suppressing it, that's an easy fix I think

Reply:
Same at other places/classes

Reply:
That is a valid option.

Reply:
What's valid option? Can't you just resolve this issue by declaring class as following rather than suppressing it?
public final class CallbackRespHelper